src/App.vue

```diff
@@ -54,12 +54,14 @@ function refreshPageTitle() {
         @pointerdown="withOutline = false"
     >
         <router-view />
-        <ErrorWindow
-            v-if="errorText"
-            title="error"
-            @close="store.dispatch('setErrorText', { errorText: null, ...dispatcher })"
-            ><div>{{ i18n.t(errorText) }}</div></ErrorWindow
-        >
+        <div>
```

Comment:
Do we need to add a `<div>` here ?

Reply:
The error window throws an error and cannot reappear afterwards if not though I don't know why exactly it needs the div.